index.js

```diff
@@ -711,6 +779,11 @@
     return Array$prototype$equals.call(this, other);
   }
 
+  //  Arguments$prototype$lte :: Arguments ~> Arguments -> Boolean
+  function Arguments$prototype$lte(other) {
+    return Array$prototype$lte.call(this, other);
+  }
+
   //  Error$prototype$toString :: Error ~> () -> String
   function Error$prototype$toString() {
```

Comment:
I think we could wait. I feel the same way about `RegExp`.

@davidchambers what are your thoughts?